### PR TITLE
DB Super ED - Far Away

### PR DESCRIPTION
**Pull request by aashutoshrathi**

### CODE REVIEW

File: js/data.json [hunk 1]
```diff
@@ -986,6 +986,12 @@ osts = [
 		"link" : "http://66.90.93.122/ost/naruto-shippuden-original-soundtrack-1/huxtigyycv/22-tragic.mp3",
 		"name" : "Naruto Shippuden OST - Tragic",
 		"img" : "http://image.noelshack.com/fichiers/2017/07/1487083218-naruto-sasuke-looses-arms.jpg"
+	},
+
+	{
+		"link" : "http://www.youtubeinmp3.com/download/get/?i=lhdrk5Is9Ksx9ziSHVf9e0iwzpQzUyX4&e=10&v=IK1K9O3y-gk",
```
Comment (anshumanv) [inline js/data.json]: Doesn't return valid mp3 response, use `fetch` instead of get from `youtubeinmp3` API please

**Comment by DivyeshPuri** [comment]
Add image which is more relatable

**Comment by anshumanv** [comment]
Is it not relatable @Lucifer24 ? I'm not sure cuz I haven't seen it.
Idea is to have some instance from anime somewhere around that music is played.

**Comment by Monal5031** [comment]
Btw @Anshuman-Verma imgur images get archived after some time, happened with me recently shouldn't we use something more permanent?

**Comment by aashutoshrathi** [comment]
Yup, I tried but no relatable (i.e. during ED) images were HD, It would be nice to share if someone has relatable ones.

**Comment by anshumanv** [comment]
If not found during ED then put any awesome instance from anime that you like @aashutoshrathi.

**Comment by aashutoshrathi** [comment]
Oh!, sorry I forgot this one is 100% relatable as this is from ED.

**Comment by DivyeshPuri** [comment]
Checkout this [link](https://images.alphacoders.com/764/thumb-1920-764254.png)

**Comment by aashutoshrathi** [comment]
@Lucifer24 Already used.

**Comment by anshumanv** [comment]
@Lucifer24 noob.

**Comment by anshumanv** [comment]
@Monal5031 I didn't know about imgur archiving, I'll probably change other instances if I find an image link broken. Thanks for bringing that to notice anyways. 👍

**Comment by Monal5031** [comment]
@Anshuman-Verma sure you'd find something in coala to use it against the link testing. ;)

**Comment by anshumanv** [comment]
Oh, then I'll incorporate it shortly.
I think travis provides a link testing too ?

**Comment by aashutoshrathi** [comment]
Yes, it do.

**Comment by aashutoshrathi** [comment]
@Lucifer24 Thanks for making convo out of this PR, this will sure as hell help us in getting points 😄

**Comment by anshumanv** [comment]
@aashutoshrathi now update song link too 😅 leave the image as it is, will update if we find a better alternative.

**Comment by aashutoshrathi** [comment]
I think I already did 😕

**Comment by anshumanv** [comment]
Try running it in the app, still returns an invalid response @aashutoshrathi

**Comment by DivyeshPuri** [comment]
@aashutoshrathi Always here for help bro

**Comment by aashutoshrathi** [comment]
Oh!!, fixed it @Anshuman-Verma

**Pull request merged**